dcgm/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,14 @@
+name: dcgm
+files:
+- name: dcgm.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/openmetrics
+  - template: instances
+    options:
+    - template: instances/openmetrics
```

Comment:
@FlorentClarret just to double-check, do you think we need any other templates here?

Reply:
nope but we could probably override the `openmetrics_endpoint` default value to show the default port the exporter is using by default

Reply:
it will be added automatically running the validate models command!

Reply:
✅ https://a.cl.ly/QwuwKKoo